ceph-dev-pipeline/build/Jenkinsfile

```diff
@@ -544,7 +562,9 @@ pipeline {
                   )
                   def ceph_builder_tag = "${env.SHA1[0..6]}.${env.BRANCH}.${env.DIST}.${ARCH}"
                   def bwc_command_base = "python3 src/script/build-with-container.py --image-repo=${env.CEPH_BUILDER_IMAGE} --tag=${ceph_builder_tag} -d ${DIST} --ceph-version ${env.VERSION}"
-                  bwc_command = "${bwc_command_base} -e custom -- rpmbuild -bb --define \\'_topdir /ceph/rpmbuild\\' /ceph/rpmbuild/SPECS/ceph-release.spec"
+                  // TODO: Backport, test, and remove --rpm-no-match-sha: https://github.com/ceph/ceph/pull/65156
+                  def rpm_no_match_release_sha = (env.RELEASE_BUILD?.toBoolean()) ? "--rpm-no-match-sha" : ""
```

Comment:
You ought to be able to remove the --rpm-no-match-sha option now as the sprm discovery code in BWC has been updated, however leaving it here for now should be harmless. However, I have deprecated this option and while I have no plan to remove it now... it might be easier to change this prior to merge and never have to worry about it getting eventually removed.